llvm/docs/GettingInvolved.rst

```diff
@@ -392,6 +401,59 @@ bot will update the channel with a link to a build bot when a build goes from
 passing to failing and again when the build goes from failing back to passing.
 It is a great way to actively monitor the status of the build.
 
+The bot also supports @mention-ing you when your email appears on a blamelist.
+For more details, DM ``help`` to the bot.
+
+.. _discord-bot-event-pings:
+
+Discord bot event pings
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Our Discord bot supports automatically sending messages about upcoming events on
+Discord. This behavior is controlled on a per-event basis, by metadata in the
```

Comment:
Maybe this paragraph should also state that the bot looks for event data in the LLVM Community calendar, as a hint for where people would have to make sure their event shows up?

Reply:
Good thought, done